controllers/namespace/controller.go

```diff
@@ -19,15 +23,76 @@ type NamespaceReconciler struct {
 	Registry string
 }
 
+var excludedNamespaceList = []string{
+	// System namespaces
+	"istio-system",
+	"kube-node-lease",
+	"kube-public",
+	"kube-system",
+	"skiperator-system",
+	"config-management-system",
+	"config-management-monitoring",
+	"asm-system",
+	"anthos-identity-service",
+	"binauthz-system",
+	"cert-manager",
+	"gatekeeper-system",
+	"gke-connect",
+	"gke-system",
+	"gke-managed-metrics-server",
+	"resource-group-system",
+	// Bundles NetworkPolicies already
+	"kasten-io",
+	// TODO needs NetworkPolicies/Skiperator
+	"vault",
+	// TODO PoC, add NetworkPolicies after
+	"sysdig-agent",
+	"sysdig-admission-controller",
+	"instana-agent",
+	"kubecost",
+	"argocd",
+	"crossplane-system",
+	"upbound-system",
+	"instana-autotrace-webhook",
+	"fluentd", //POC
+}
+
+func (r *NamespaceReconciler) isExcludedNamespace(ctx context.Context, namespace string) bool {
+	configMapNamespacedName := types.NamespacedName{Namespace: "skiperator-system", Name: "namespace-exclusions"}
+	namespaceExclusionCMap := corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configMapNamespacedName.Name,
+			Namespace: configMapNamespacedName.Namespace,
+		},
+	}
+
+	err := r.GetClient().Get(context.Background(), configMapNamespacedName, &namespaceExclusionCMap)
+	if errors.IsNotFound(err) {
+		err = r.GetClient().Create(ctx, &namespaceExclusionCMap)
+		if err != nil {
+			fmt.Printf("Something went wrong creating namespace-exclusion config map: %v", err.Error())
+			return true
+		}
+	}
+	if err != nil {
+		fmt.Printf("Something went wrong getting namespace-exclusion config map: %v", err.Error())
+		return true
+	}
```

Comment:
should we return true or false upon errors here? should we maybe use `panic()` instead?

Reply:
I think this is a panic, as it is impossible for skiperator to continue if it does not know what namespaces are excluded